lib/ddtrace/contrib/rails/framework.rb

```diff
@@ -47,6 +48,10 @@ def self.configure(config)
             port: datadog_config[:trace_agent_port]
           )
 
+          # set default tracer tags
+          datadog_config[:tracer].set_tags(datadog_config[:tags])
+          datadog_config[:tracer].set_tags('env' => datadog_config[:env])
```

Comment:
In think we're almost there, but what happens here if `:env` is not defined it the conf at all? Could this happen and override a legit `:tags` value?